integration-tests/js-compute/fixtures/byte-repeater/fastly.toml

```diff
@@ -13,4 +13,4 @@ service_id = ""
   [local_server.backends]
 
     [local_server.backends.TheOrigin]
-      url = "https://compute-sdk-test-backend.edgecompute.app/"
+      url = "JS_COMPUTE_TEST_BACKEND/"
```

Comment:
The `JS_COMPUTE_TEST_BACKEND` value gets replaced before the test is run, which allows us to re-target the test to a local backend.